leader-election-impl/src/main/java/com/palantir/paxos/PaxosConstants.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.paxos;
+
+public final class PaxosConstants {
+
+    public static final boolean CANCEL_REMAINING_CALLS = false;
```

Comment:
if we're just makign this be a constant, please can we just remove the optionality?

Reply:
This simply expands the existing constant. I agree that the optionality should be removed, but this is causing serious problems at the moment and must be addressed quickly.